de/px-emails.json

```diff
@@ -217,7 +217,7 @@
                         "thatsIt": "Das war's. Klicke einfach 3 Minuten bevor deine Session beginnt…"
                     },
                     "join": "Session beitreten",
-                    "alternativeLink": "Nur für den Fall, dass der Button oben nicht funktioniert ... hier ist der Link: {{confirmLink}}",
+                    "alternativeLink": "Nur für den Fall, dass der Button oben nicht funktioniert ... hier ist der Link: {{sessionLink}}",
```

Comment:
Damn, not again... I think that `sessionLink` is the correct one, can you confirm with the English translation just in case?

Reply:
It should be indeed [sessionLink](https://github.com/IntelliZoomPlatform/communication-service/blob/master/lib/subscriptions/session-starts-soon.js#L75) and unless I'm mistaken, this issue is happening in English too: https://github.com/IntelliZoomPlatform/px-translations/blob/master/en/px-emails.json#L220

Reply:
Could it be this? 
![image](https://user-images.githubusercontent.com/87361014/164651720-a4e5f4c2-ac13-458e-81ba-49c74fbea9d3.png)

Reply:
Moving it with ux

Reply:
@jcagigal-uz

Reply:
All good, confirmed with UX. Will wait until Monday to merge regardless.